### PR TITLE
Fixes #891 - Adds the leanplum sdk

### LINKED ISSUE #891: Add leanplum sdk

### PR DESCRIPTION
**Pull request by boek**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -73,3 +73,7 @@ gen-external-apklibs
 
 # macOS
 .DS_Store
+
+# Token files
+.leanplum_token
+.adjsut_token
```
Comment (pocmo) [inline .gitignore]: adj**sut**

File: app/build.gradle [hunk 2]
```diff
@@ -202,6 +202,25 @@ android.applicationVariants.all { variant ->
...
+        def key = parts[1]
+        buildConfigField 'String', 'LEANPLUM_ID', '"' + id + '"'
+        buildConfigField 'String', 'LEANPLUM_TOKEN', '"' + key + '"'
+        println "(Added from .leanplum_token file)"
+    } catch (FileNotFoundException ignored) {
+        buildConfigField 'String', 'LEANPLUM_ID', 'null'
+        buildConfigField 'String', 'LEANPLUM_TOKEN', 'null'
+        println("X_X")
+    }
 }
```
Comment (pocmo) [inline app/build.gradle]: Isn't the code missing that actually takes the token (from taskcluster secrets?) and places it in this file?

Comment (boek) [reply app/build.gradle]: Following up here  #896. My bad I didn't do a great job creating all the sub issues before starting the work. With the move towards breaking tasks up I'm trying to have smaller branches 👍

File: .gitignore (hunk 1 above)
Comment (boek) [reply .gitignore]: good catch!

**Comment by boek** [comment]
@pocmo It wasn't apart of the initial ask, but I wouldn't be surprised! I will create a follow up issue for triage

**Pull request merged**